### DISCUSSION TITLE
Question about special characters and same album names

### DISCUSSION THREAD

**Discussion by ghost** [jojo2357/Music-Discord-Rich-Presence#22] [Q&A] [answered]
Can I just put the exact album name in the .dat file even with special characters like slash, comma, period, or equal sign?

Also, in the unfortunate case where two albums have the same name, what happens then?

**Comment by jojo2357** [comment, marked as answer]
If you only want the format of 

```
music.ui=groove
id=812345678910
album
otheralbum
```

where each album is on its own line, you have to convert it to all lowercase and remove special chars the way it appears in discord as specified in the readme. Otherwise you can use the other formats.

If you want to have multiple arts for the same album, the only way this is doable if you have multiple players. If this is an issue that needs to be fixed then I really don't know how to help. Im assuming you have albums with identical names from different artists? maybe I could add an artist specification? I dunno seems like it could only get clunky and I dunno how common this issue would be :|

**Reply by jojo2357** [reply]
oh and there are no "special characters" or "escape characters" so its all just strings btw slashes get no special treatment

**Reply by ghost** [reply]
I see, I'm starting to get it now.

I don't have any albums with same names. I was just wondering what would happen or what I need to do if that time comes.

I'm really grateful for your work and support. Thanks! :)

**Reply by jojo2357** [reply]
of course! feel free to share it would mean a lot :D
